docs/pages/x/api/date-pickers/date-calendar.json

```diff
@@ -14,19 +15,25 @@
     "disablePast": { "type": { "name": "bool" } },
     "displayWeekNumber": { "type": { "name": "bool" } },
     "fixedWeekNumber": { "type": { "name": "number" }, "default": "undefined" },
+    "focusedView": {
```

Comment:
`DateCalendar` already had this prop
No idea why the doc gen was not getting it
Same for `onFocusedViewChange`